src/www/interfaces_vlan_edit.php

```diff
@@ -200,6 +206,31 @@
                       </div>
                     </td>
                   </tr>
+<?
+$priorities = array(
+    1   => gettext('1 - Background'),
+    0   => gettext('0 - Best Effort (default)'),
```

Comment:
Actually, it doesn't make sense to put the default value first in this case, since they are in order of priority, 1 - Backgroud is lower priority than 0 - Best Effort